tools/ds-identify

```diff
@@ -281,6 +280,49 @@ read_fs_info() {
     DI_ISO9660_DEVS="${isodevs#,}"
 }
 
+read_geom() {
+    local oifs="$IFS" line="" delim=","
+    local ret=0 out="" labels="" dev="" label="" ftype="" isodevs="" uuids=""
+    out=$(geom label status -as) || {
+        ret=$?
+        error "failed running [$ret]: geom label status -as"
+        DI_FS_LABELS="$UNAVAILABLE:error"
+        DI_ISO9660_DEVS="$UNAVAILABLE:error"
+        return $ret
+    }
+
+    # shellcheck disable=2086
+    { IFS="$CR"; set -- $out; IFS="$oifs"; }
+
+    for line in "$@"; do
```

Comment:
i'd like to see an example of what these lines look like.

Reply:
```
meena@fbsd12-1 /u/h/meena> geom label status -a
          Name Status Components
  gpt/gptboot0 N/A vtbd1p1
     gpt/swap0 N/A vtbd1p2
iso9660/cidata N/A vtbd2
```

and here, in -s for script-friendly output:

```
meena@fbsd12-1 /u/h/meena> geom label status -as
  gpt/gptboot0 N/A vtbd1p1
     gpt/swap0 N/A vtbd1p2
iso9660/cidata N/A vtbd2
```